basics/exceptions.md

```diff
@@ -0,0 +1,142 @@
+# Exceptions
+
+Dieser Abschnitt befasst sich nur mit User-`Exceptions` - System-`Error` sind 
+normalerweise fatal und sollten __nie__ auffangen werden.
+
+### Exceptions auffangen
+
+Ein Anwendungsfall für Exceptions ist die Validierung von potentiell invaliden
+Benutzereingaben. Sobald eine Exception geworfen wurde, wird der Stack abgewickelt
+(engl: to unwind), bis der erste passende Exception-Handler gefunden wurde.
+
+```d
+try
+{
+    readText("dummyFile");
+}
+catch (FileException e)
+{
+    // ...
+}
+```
+
+Die Verwendung mehrerer `catch`-Blocks ist möglich. Ein abschließender
+`finally`-Block wird unabhängig von einem Fehlerauftreten ausgeführt.
+Exceptions werden mit`throw` geworfen.
+
+```d
+try
+{
+    throw new StringException("You shall not pass.");
+}
+catch (FileException e)
+{
+    // ...
+}
+catch (StringException e)
+{
+    // ...
+}
+finally
+{
+    // ...
+}
+```
+
+Beachte: Die Verwendung eines [scope guards](gems/scope-guards) ist 
```

Comment:
Ah - du hast die Dateien noch nicht zur `index.yml` hinzugefügt. Ohne werden sie nicht in der Tour angezeigt. Für mehr Infos:
https://github.com/dlang-tour/german/blob/master/basics/index.yml